### DISCUSSION TITLE
Bypass the 50MB file size limit

### DISCUSSION THREAD

**Discussion by Felix14-v2** [nlef/moonraker-telegram-bot#324] [Ideas]
Hello everyone, and thanks for such an amazing tool! I found recently that developers can use a self-hosted (that's what "local" mean, right?) API server to extend the file limit up to 2000 megabytes, [^1] so here I have some questions about it:
- Can an ordinary user with no programming experience use this to bypass the 50MB file size limit for sending timelapses? (And how if that's possible)
- Can that solution be integrated into the bot installation process?

I may be fundamentally misunderstanding something, so, please, be patient 🙂
[^1]:https://core.telegram.org/bots/api#using-a-local-bot-api-server

**Comment by thankjura** [comment]
See https://github.com/nlef/moonraker-telegram-bot/pull/341
